lib/dialogs/import/dropzone/style.scss

```diff
@@ -1,37 +1,86 @@
 .importer-dropzone {
   display: flex;
+  cursor: pointer;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  padding: 24px;
+  padding: 16px;
   border-width: 2px;
   border-style: dashed;
-  background: $studio-simplenote-blue-5;
   transition: padding 0.2s ease-out, background 0.2s ease-out;
+  text-align: center;
+  color: $studio-gray-50;
+  background-color: transparent;
 
   &:focus-within {
     overflow: hidden;
-    outline: $focus-outline;
-  }
-
-  @at-root .theme-dark & {
-    background: $studio-gray-70;
+    outline: none;
```

Comment:
IMO it is a poor practice to disable `outline` styles for interactive elements. Keyboard users rely upon these styles to track focus selection in the UI.

In my experience, browsers have gotten better about not applying outlines to elements purely because they were clicked, which I believe is why many people tend to disable `outline`. Regardless, I believe creating a product with an accessible design benefits all and I view focus outlines as a postive.

What do you think about not setting `outline: none`?

Reply:
I have no issues with it no. I have removed that line now.